packages/aws-cdk/README.md

```diff
@@ -162,6 +162,57 @@ $ cdk deploy --parameters "ParametersStack:TopicNameParam=blahagain" --force
 
 ⚠️ Parameters do not propagate to NestedStacks. These must be sent with the constructor. See Nested Stack [documentation](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-cloudformation.NestedStack.html)
 
+##### Outputs
+
+Write stack outputs from deployments into a file. When your stack finishes deploying, all stack outputs will be written to the output file as JSON.
+
+Usage of output in a CDK stack
+```typescript
+const fn = new lambda.Function(this, "fn", {
+      handler: "index.handler",
+      code: lambda.Code.fromInline(`exports.handler = \${handler.toString()}`),
+      runtime: lambda.Runtime.NODEJS_10_X
+    });
+
+    new cdk.CfnOutput(this, 'FunctionArn', {
+      value: fn.functionArn,
+    });
+```
+
+Specify an outputs file to write to by supplying the `--outputs-path` parameter
+
+```console
+$ cdk deploy --outputs-path "/Users/code/myproject/outputs.json"
+```
+
+When the stack finishes deployment, `outputs.json` would look like
+```json
+{
+  "MyStack": {
+    "FunctionArn": "arn:aws:lambda:us-east-1:123456789012:function:MyStack-fn5FF616E3-G632ITHSP5HK"
```

Comment:
The name of the output here will not be `FunctionArn` but rather the logical id of the output, no?

Reply:
I'm using CfnOutput in this example. 

```typescript
new cdk.CfnOutput(this, 'FunctionArn', {
      value: fn.functionArn,
    });
```
They're maintained as they're named in my app. If there are conflicts, we end up with an error. 
Higher level constructs that provide stack outputs out of the box *would* contain the logical ID as the Key

does that answer your question?

Reply:
Yes, got it. I just thought that the example could lead to think that the key is the id while it's the logical id (and in this case both are the same).

Reply:
good callout, i'll see if i can disambiguate further by mentioning logical id explicitly